packages/dbgpt-app/src/dbgpt_app/openapi/api_v1/attachment_react_adapter.py

```
@@ -0,0 +1,608 @@
+    ``steps`` and ``final_content`` additionally get known server path
+    prefixes (home directory, registry work root) replaced with
+    ``<server-path>``: materialized paths legitimately surface in tool
+    observations and tracebacks during a turn, and the share view is the
+    trust boundary where they are masked. ``generated_images``,
+    ``sub_agents`` and ``citations`` are passed through unchanged.
+    """
+    if not isinstance(context, str):
+        return context
+    try:
+        payload = json.loads(context)
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)